elastalert/zabbix.py

```diff
@@ -0,0 +1,102 @@
+from alerts import Alerter  # , BasicMatchString
+import logging
+# from pprint import pprint
+from pyzabbix.api import ZabbixAPI
+from pyzabbix import ZabbixSender, ZabbixMetric
+# from pyzabbix.api import ZabbixAPIException
+# from requests.packages.urllib3.exceptions import InsecureRequestWarning
```

Comment:
Can you remove all the commented out imports?

Reply:
Removed